client/e2e/support/user-list-po.ts

```diff
@@ -26,15 +26,12 @@ export class UserListPage {
   }
 
   changeView(viewType: 'card' | 'list') {
-    // in the test of changing the view, .check will not work if you look for mat-radio-button
-    // or, basically anything other than :radio and it will never be ready to click,
-    // so you have to use force: true as an option to not wait for it to be ready
-    return cy.get(`:radio[value="${viewType}"]`).check({ force: true }).click({ force: true });
+    return cy.get(`#view-type-radio .mat-radio-button[value="${viewType}"]`).click();
```

Comment:
The idea here is just that because of how Angular Material wraps the `input` element its hidden in such a way that a user couldn't actually directly interact with it so Cypress doesn't want to let us either. It seems like a better idea to just click the `.mat-radio-button` itself.